plugins/resizimg/trumbowyg.resizimg.js

```diff
@@ -38,7 +38,7 @@
                                     return false;
                                 },
                                 onDragEnd: function () {
-                                    trumbowyg.$c.trigger('tbwchange');
+                                    trumbowyg.$c.trumbowyg('execCmd', {cmd: 'syncCode'});
```

Comment:
From here, you should call `trumbowyg.syncCode()` since we have direct access to the internal API :)

Reply:
Sure, no prob.